test/clj/chronograph/domain/task_test.clj

```diff
@@ -0,0 +1,144 @@
+(ns chronograph.domain.task-test
+  (:require [clojure.test :refer :all]
+            [clojure.spec.alpha :as s]
+            [clojure.spec.gen.alpha :as gen]
+            [chronograph.db.core :as db]
+            [chronograph.db.task :as db-task]
+            [chronograph.domain.task :as task]
+            [chronograph.factories :as factories]
+            [chronograph.fixtures :as fixtures]
+            [chronograph.utils.time :as time]
+            [next.jdbc :refer [with-transaction]]))
+
+(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-task-test
+  (let [{user-id :users/id} (factories/create-user)
+        {organization-id :organizations/id} (factories/create-organization user-id)]
+    (testing "Can create task with name and description"
+      (with-redefs [time/now (constantly (time/now))]
+        (let [created-task (task/create {:name "Foo"
+                                         :description "Description of Foo"
+                                         :organization-id organization-id})
+              retrieved-task (task/find-by-id (:tasks/id created-task))
+              now (time/now)]
+          (is (= #:tasks{:id (:tasks/id created-task)
+                         :name "Foo"
+                         :description "Description of Foo"
+                         :organization-id organization-id
+                         :created-at now
+                         :updated-at now
+                         :archived-at nil}
+                 retrieved-task)))))
+    (testing "It throws an error if name is not present"
+      (is (thrown? org.postgresql.util.PSQLException
```

Comment:
Can we do a more specific assert? Like null constraint? PSQLException is too broad I feel.